python-regression/util/threading_logic/thread_logic.py

```diff
@@ -2,29 +2,28 @@
 from aloe import world
 
 
-'''
-Places the variables into a queue for threadsafe access
-
-@param args:    Variables to be placed in queue, in the order they should be called on 
-@return         Returns a populated queue
-'''
 def populate_queue(*args):
+    """
+    Places the variables into a queue for threadsafe access
+
+    :param args: Variables to be placed in queue, in the order they should be called on
+    :returns q: Returns a populated queue
+    """
     q = queue.Queue()
-    for arg in range(len(args)):
+    for arg in enumerate(args):
```

Comment:
`arg` is an index?
`for arg in args:`

Anyhows you get the idea. continue changing this